tests/reversible_graph_net.py

```diff
@@ -0,0 +1,138 @@
+import unittest
+
+import torch
+import torch.nn as nn
+DEVICE = torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')
+
+import sys
+sys.path.append('../')
+from FrEIA.modules import *
+from FrEIA.framework import *
+
+
+def F_conv(cin, cout):
+    '''Simple convolutional subnetwork'''
+    return nn.Sequential(nn.Conv2d(cin, 32, 3, padding=1),
+                         nn.ReLU(),
+                         nn.Conv2d(32, cout, 3, padding=1))
+
+def F_fully_connected(cin, cout):
+    '''Simple fully connected subnetwork'''
+    return nn.Sequential(nn.Linear(cin, 128),
+                         nn.ReLU(),
+                         nn.Linear(128, cout))
+
+
+class ComplexComputeGraph(unittest.TestCase):
+
+
+    def __init__(self, *args):
+        super().__init__(*args)
+
+        self.inp_size = (3, 10, 10)
+        self.cond_size = (1, 10, 10)
+
+        inp = InputNode(*self.inp_size, name='input')
+        cond = ConditionNode(*self.cond_size, name='cond')
+
+        split = Node(inp,
+                     Split,
+                     {'split_size_or_sections': [1,2], 'dim': 0},
```

Comment:
Parameter is called `section_sizes` now :)

Reply:
Ah, I think that's someone else's department, I don't know why it's in my PR.
But I will change it nonetheless! :smile: